### ISSUE TITLE
Bug: Call doesn't end for both participants

### ISSUE THREAD

**Issue by renatokeys** [project-error/npwd#332]
### Developement Issue

Yes

### Issue Description

Calls doesnt working correctly on vrp base.

When player end the current call a promisse event npwd:endCall got status ok and user1 phone disconnected call, but the user2 continue call and when him click on endcall get error, the call doesnt exist anymore.

### Issue Reproduction

1. Login in 2 users
2. Make a call
3. End call
$. Who clicked will got success and disconnect call, but other contact doesnt update call status

### Evidence

![unknown (2)](https://user-images.githubusercontent.com/6384274/137758132-853f9903-8fb8-4154-9ecc-20449003f724.png)

**Comment by RockySouthpaw** [comment]
Reproduced so not a vrp issue.

Whats actually happening here is that:
```ts
    if (currentCall?.is_accepted) {
      emitNet(CallEvents.WAS_ENDED, currentCall.receiverSource);
    }
 ```
 This is only being triggered for the receiver on the end call event. It should also be triggering for the transmitterSource too kind of like so:
 
 ```ts
 if (currentCall?.is_accepted) {
      emitNet(CallEvents.WAS_ENDED, currentCall.receiverSource);
      emitNet(CallEvents.WAS_ENDED, currentCall.transmitterSource);
    }
```

This will resolve the issue, however, there is another issue that also exist where if the call initiator ends the call before the receiver has a chance to answer, the receiver is unable to end their call.

Call logic probably needs to be refactored to where if the call state is still ringing then it needs to end the call for the receiver too.

**Comment by TasoOneAsia** [comment]
Fixed with #374
